packages/cpp/ArmoniK.Api.Client/source/submitter/SessionContext.cpp

```diff
@@ -0,0 +1,11 @@
+#include "submitter/SessionContext.h"
+
+SessionContext::SessionContext(armonik::api::grpc::v1::TaskOptions task_options) : task_options_(
+    std::move(task_options))
+{
+}
+
+void SessionContext::set_session_id(const std::string& session_id)
+{
+  session_id_ = session_id;
+}
```

Comment:
These should be defined in the header to enable inlining

Reply:
I removed the class